internal/idle/idle_e2e_test.go

```diff
@@ -259,15 +264,14 @@ func (s) TestChannelIdleness_Enabled_OngoingCall(t *testing.T) {
 				}
 
 				// Verify that there are no idleness related channelz events.
+				//
+				// TODO: Improve the checks here. If these log strings are
+				// changed in the code , these checks will continue to pass.
```

Comment:
Remove the space before the comma please.  (Never know what other linters will think later unfortunately.)

Reply:
Done.